### PR TITLE
Add RHOSP multi-pod initial changes

### PR DESCRIPTION
**Pull request by wangxian-ibm**
Add automation to support mulit-pod deployment for RHOSP

### CODE REVIEW

File: roles/common/tasks/networking.yml [hunk 1]
```diff
@@ -91,22 +91,38 @@
 - name: Check if interface has its own config file
```
Comment (nirajdp76) [inline roles/common/tasks/networking.yml]: can you group below os == 'ubuntu' tasks under one block with when: os == 'ubuntu'

File: roles/common/tasks/networking.yml [hunk 2]
```diff
@@ -91,22 +91,38 @@
...
 
 #Check if the static routes are present in interface configuration file, so that they are persistant across reboots.
 - name: Check for static routes in {{ nw_config_file }}
   command: "grep -h 'ip route add {{ item.cidr }} via {{ nw_gw }}' {{ nw_config_file }}"
   with_items: "{{ undercloud_cidr }}"
   changed_when: False
-  when: (update_routes|default('false')|bool) and (not ( ansible_default_ipv4.network  == ( item.cidr | ipaddr('network') )))
+  when: (update_routes|default('false')|bool) and (not ( ansible_default_ipv4.network  == ( item.cidr | ipaddr('network') ))) and os == 'ubuntu'
+
+- name: Check if route configuration exists
```
Comment (nirajdp76) [inline roles/common/tasks/networking.yml]: can you group below os == 'rhel' tasks under one block with when: os == 'rhel'

**Comment by nirajdp76** [comment]
ok to test

**Comment by wangxian-ibm** [comment]
retest

**Comment by wangxian-ibm** [comment]
test

**Comment by wangxian-ibm** [comment]
Updated

**Comment by nirajdp76** [comment]
retest

**Comment by nirajdp76** [comment]
ok to test

**Pull request merged**